sdk/master_board_sdk/include/master_board_sdk/defines.h

```diff
@@ -32,13 +31,15 @@ struct imu_packet_t
 	int16_t accelerometer[3];
 	int16_t gyroscope[3];
 	int16_t attitude[3];
+	int16_t linear_acceleration[3];
 } __attribute__((packed));
 
 struct imu_data_t
 {
 	float accelerometer[3];
 	float gyroscope[3];
 	float attitude[3];
+	float linear_acceleration[3];
```

Comment:
What's the difference between `accelerometer` and `linear_acceleration`?

Reply:
It is gravity compensated. This computation is done internally by the KF on the IMU. This will be particularly useful for the velocity estimator I'm working on.

Reply:
Mhmm, `acceleration_without_gravity` would be a nice name but a bit length. 

If `linear_acceleration` is what people call it usually, then that's fine with me.